NachoSpigot-Server/src/main/java/net/minecraft/server/DedicatedServer.java

```diff
@@ -459,7 +455,15 @@ public boolean ae() {
 
     public boolean ai() {
         // [Nacho-0039] Add a check to see if we are using Linux or not, if not ignore this.
-        return this.propertyManager.getBoolean("use-native-transport", true) && org.apache.commons.lang.SystemUtils.IS_OS_LINUX;
+        return this.getTransport() == ServerConnection.EventGroupType.EPOLL && org.apache.commons.lang.SystemUtils.IS_OS_LINUX;
+    }
+
+    public ServerConnection.EventGroupType getTransport() {
+        try {
+            return ServerConnection.EventGroupType.valueOf(this.propertyManager.getString("transport-to-use", "default").toUpperCase());
+        } catch (Exception ignored) {
+            return ServerConnection.EventGroupType.DEFAULT;
+        }
```

Comment:
I feel like the `use-native-transport` option should stay as it was, and use KQueue or Epoll depending on what platform the server is running on.

Reply:
Normally its set to default, this then chooses if it should use Epoll, KQueue, or NIO if none of the others are supported. Why should we add that option back, if it's still there but now you need to do:
`transport-to-use: epoll` for epoll
`transport-to-use: kqueue` for kqueue
`transport-to-use: default` for epoll, kqueue, or NIO depending on your system

Reply:
This is from the default Minecraft `server.properties`, correct? I don't feel like we should be modifying the structure upstream configs
Also, I've never seen situations where both KQueue and Epoll are supported at the same time, so I'm not sure how useful the option is

Also, should probably add NIO as an `EventGroupType` if it does stay, as it doesn't look like there's a way to explicitly select it

Reply:
I'll add something for that 👍 Please see the new PR where I continue #104